articles/machine-learning/resource-azure-container-for-pytorch.md

```diff
@@ -43,14 +43,13 @@ Azure Container for PyTorch is a lightweight, standalone environment that includ
 
 The following configurations are supported:
 
-| Environment Name | OS | GPU Version| Python Version | PyTorch Version | ORT-training Version | DeepSpeed Version | torch-ort Version |
-| --- | --- | --- | --- | --- | --- | --- | --- |
-|acpt-pytorch-2.0-cuda11.7|Ubuntu 20.04|cu117|3.8|2.0|1.14.1|0.8.2 |0.14.0|
-|acpt-pytorch-1.13-cuda11.7|Ubuntu 20.04|cu117|3.8|1.13.1|1.14.1|0.8.2|1.14.0|
-|acpt-pytorch-1.12-py39-cuda11.6|Ubuntu 20.04|cu116|3.9|1.12.1|1.14.1|0.8.2|1.14.0|
-|acpt-pytorch-1.12-cuda11.6|Ubuntu 20.04|cu116|3.8|1.12.1|1.14.1|0.8.2|1.14.0|
-|acpt-pytorch-1.11-cuda11.5|Ubuntu 20.04|cu115|3.8|1.11.0|1.11.1|0.7.3|1.11.0|
-|acpt-pytorch-1.11-cuda11.3|Ubuntu 20.04|cu113|3.8|1.11.0|1.14.1|0.8.2|1.14.0|
+| Environment Name | OS | GPU Version| Python Version | PyTorch Version | ORT-training Version | DeepSpeed Version | torch-ort Version | Nebura Version | 
```

Comment:
Nebula versions are not needed. But if added there is typo- it should be "Nebula".